git-try-push/main.js

```diff
@@ -38,7 +39,7 @@ async function main() {
             try {
                 // Try to push, if successful, then checkout previous branch and just exit.
                 // Don't try to force push the first time in case it's not necessary.
-                if (force && i>0)
+                if ((forceOverwrite && branch != "master") || (force && i>0))
```

Comment:
Might want to add `main` in here (and anywhere else it's used?) for future-proofing too?